File: VB/DevExpress.Xpf.DnD/DragAndDrop/DragAndDropDataContainer.vb
Imports Microsoft.VisualBasic
Imports System

Namespace DX.Xpf.DnD
	Public Class DragAndDropDataContainer
		Private _dragData As Object
		Public Property DragData() As Object
			Get
				Return _dragData
			End Get
			Set(ByVal value As Object)
				_dragData = value
			End Set
		End Property
	End Class
End Namespace

File: CS/DevExpress.Xpf.DnD/DragAndDrop/DragAndDropDataContainer.cs
using System;

namespace DX.Xpf.DnD {
    public class DragAndDropDataContainer {
        private object _dragData;
        public object DragData {
            get {
                return _dragData;
            }
            set {
                _dragData = value;
            }
        }
    }
}
